R/prepareOutputForNullModel.R

```diff
@@ -29,10 +29,19 @@
     zeroFLAG <- NULL
     RSS <- ifelse(family$family == "gaussian", sum(resid.marginal^2)/varComp/(nrow(X) - ncol(X)), 1)
 
+    # Sample-level data frame.
+    fit <- data.frame(
+      outcome = as.vector(y),
```

Comment:
Is it weird that we mix `outcome` and `workingY`? Should it be `Y` and `workingY` or `outcome` and `workingOutcome`? Maybe I'm overthinking.

Reply:
I think either keep it as is or change `workingY` to `workingOutcome`. The `workingY` element is used in a lot of places right now.

Reply:
Leaning towards keeping as is unless it's too confusing just based on time.